config/storage.yml

```diff
@@ -8,10 +8,10 @@ local:
 
 amazon:
   service: S3
-  access_key_id: <%= Rails.application.secrets.aws_access_key_id %>
-  secret_access_key: <%= Rails.application.secrets.aws_secret_access_key %>
-  region: <%= ENV.fetch("AWS_REGION", "eu-west-3") %>
-  bucket: <%= ENV.fetch("AWS_BUCKET_NAME", "decidim-aytopinto-prod") %>
+  access_key_id: <%= Rails.application.secrets.aws.access_key_id %>
```

Comment:
`Rails.application.secrets.aws` returns a hash and we can not access the keys of a hash with `.access_key_id` for example.

Better use `Rails.application.secrets.aws["access_key_id"]